test/jobs/pause-jobs.ts

```diff
@@ -0,0 +1,59 @@
+import { expect } from 'chai';
+import { v4 as uuid } from 'uuid';
+import { JobStatus, Job } from './../../app/models/job';
+
+// unit tests for pausing/resuming jobs
+
+describe('Pausing jobs', function () {
+  describe('When a job is running', function () {
+    const requestid = uuid();
+    const runningJob = new Job({
+      jobID: requestid,
+      username: 'anonymouse',
+      requestId: requestid,
+      status: JobStatus.RUNNING,
+      request: 'foo',
+      numInputGranules: 10,
+      collectionIds: ['C123'],
+    });
+
+    describe('and it is resumed before pausing', function () {
+      it('throws an error', function () {
+        expect(runningJob.resume.bind(runningJob)).to.throw('Job is status is running - only paused jobs can be resumed.');
```

Comment:
Extra is - should be fixed in the code and then updated here.

Reply:
fixed